libPhoneNumber/NBPhoneNumberOfflineGeocoder.h

```diff
@@ -0,0 +1,40 @@
+//
+//  NBPhoneNumberOfflineGeocoder.h
+//  libPhoneNumberiOS
+//
+//  Created by Rastaar Haghi on 6/12/20.
+//  Copyright © 2020 ohtalk.me. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "NBGeocoderMetadataHelper.h"
+#import "NBPhoneNumber.h"
+#import "NBPhoneNumberDefines.h"
+#import "NBPhoneNumberUtil.h"
```

Comment:
This is not used in the header, if you need to use it in the .m file, move the import to the .m file

Reply:
Yup, moved to .m